README.md

```diff
@@ -1 +1,32 @@
-# mashov
\ No newline at end of file
+## Steps to generate the forms and email drafts for each employee.
+
+1. Clone this repository.
+
+```
+git clone https://github.com/simon3z/mashov
```

Comment:
@scollier OK you removed my name but you still reference my repo, let's assume that if you're reading this you already are familiar with git and know how to clone this repo (whatever for "this" is).